### DISCUSSION TITLE
This is so cool, and faster than YouTube Music!

### DISCUSSION THREAD

**Discussion by nikolqy** [snuffyDev/Beatbump#19] [General]
I love how this is an alternative to YouTube music but uses YouTube music content. So cool. I just wish there was a way to create playlists and put songs on repeat. Maybe there is and it just isn't working for me? I see the playlists feature but I can't find a button to add a song to a playlist.

Anyways, really cool creation and keep up the good work! I look forward to what's coming next!

**Comment by snuffyDev** [comment]
I appreciate the kind words! For awhile I’ve been planning on adding looping. Since it is only a line of code to implement, I’ve almost jumped the gun a few times and nearly implemented it. The only thing that’s holding me back is implementing it correctly in terms of UI and User Experience. Though I’ve recently had a few ideas and was gonna start experimenting with it soon. Keep an eye out!

As for adding to playlists, it’s come to my attention recently that I totally overlooked being able to add a song that’s being played to a playlist.

If you meant that in general you are unable to figure out how to do it, here’s how. On the top right of thumbnails in carousels, or on the right hand side of search results, there (at least should be) 3 white dots which will open a menu. From there you should be able to click “add to playlist” and then it’ll let you create and customize them.

The folder icon in the navbar is where you can find your library of favorites and playlists. You can also create and edit playlists from here as well.

Just in case, here’s what the menu looks like on mobile. Will add a desktop example in a minute 
https://user-images.githubusercontent.com/72365477/150850911-ce91d670-e3cb-4ba6-a8ed-efa3416d5449.MP4

**Reply by snuffyDev** [reply]
Here's what it looks like on a computer. When hovering over an item in the carousel, the menu button should show at the top right of the thumbnail. There's also the ability to use the Tab key to navigate, but it's sort of clunky since I'm still trying to figure out the best approach in terms of accessibility as well as overall user experience.

https://user-images.githubusercontent.com/72365477/150852631-de5d5f5d-3208-41f8-9131-2d0701605ffe.mp4
